package.json

```diff
@@ -50,22 +50,22 @@
   },
   "dependencies": {
     "@angular-slider/ngx-slider": "2.0.4",
-    "@angular/animations": "13.3.11",
-    "@angular/cdk": "13.3.9",
-    "@angular/common": "13.3.11",
-    "@angular/compiler": "13.3.11",
-    "@angular/core": "13.3.11",
-    "@angular/forms": "13.3.11",
-    "@angular/material": "13.3.9",
-    "@angular/material-date-fns-adapter": "13.3.9",
-    "@angular/platform-browser": "13.3.11",
-    "@angular/platform-browser-dynamic": "13.3.11",
-    "@angular/router": "13.3.11",
+    "@angular/animations": "14.0.1",
+    "@angular/cdk": "14.0.1",
+    "@angular/common": "14.0.1",
+    "@angular/compiler": "14.0.1",
+    "@angular/core": "14.0.1",
+    "@angular/forms": "14.0.1",
+    "@angular/material": "14.0.1",
+    "@angular/material-date-fns-adapter": "14.0.1",
+    "@angular/platform-browser": "14.0.1",
+    "@angular/platform-browser-dynamic": "14.0.1",
+    "@angular/router": "14.0.1",
     "@azure/msal-angular": "2.2.0",
     "@azure/msal-browser": "2.23.0",
     "@ngneat/transloco": "3.2.0",
-    "@ngrx/component-store": "13.0.2",
-    "@nrwl/angular": "14.1.9",
+    "@ngrx/component-store": "13.2.0",
```

Comment:
I can see that NGRX seems to be in release candidate of version `14`, should we create a task for that now so we remember to have an eye on that?

Reply:
I think it will naturally come up during the next dependency update. Which I will suggest is something we do regularly every x weeks to avoid spending a week updating.